README.md

```diff
@@ -100,15 +100,15 @@ pip install pandas
 ```
 
 ## Dependencies
-- [NumPy](https://www.numpy.org)
-- [python-dateutil](https://labix.org/python-dateutil)
-- [pytz](https://pythonhosted.org/pytz)
+- [NumPy - Adds support for large, multi-dimensional arrays, matrices and high-level mathematical functions to operate on these arrays](https://www.numpy.org)
+- [python-dateutil - Provides powerful extensions to the standard datetime module, available in Python 2.3+](https://labix.org/python-dateutil)
```

Comment:
you can remove the , available from datetutil and pytz

Reply:
Understood will do that, thanks for the suggestions @jreback